src/main/java/com/labs/introtoprogramming/lab4/Main.java

```diff
@@ -8,6 +8,7 @@
 import com.labs.introtoprogramming.lab4.image.io.bmp.BMPImageWriter;
 import com.labs.introtoprogramming.lab4.image.transform.RGBImageTransformation;
 import com.labs.introtoprogramming.lab4.image.transform.Scale;
+import com.labs.introtoprogramming.lab4.image.transform.SimpleBrightness;
```

Comment:
Remove this import. Brightness is not implemented yet :)

Reply:
Sure.